app/presenters/setting_presenter.rb

```diff
@@ -102,7 +102,10 @@ def select_values
   def _assign_attribute(k, v)
     if k.to_s == 'value'
       @explicit_value = !v.nil?
-      write_attribute(k, v)
+
+      name = self.class.attribute_aliases[k.to_s] || k.to_s
+      @attributes.write_from_user(name, v)
+      value
```

Comment:
I've fixed this in https://github.com/theforeman/foreman/pull/9130 which is better solution than what we have and is backwards compatible.

Reply:
It's in, can you rebase pls? :)